lms/views/predicates/_predicates.py

```diff
@@ -0,0 +1,101 @@
+def is_canvas_file(_context, request):
```

Comment:
We could use pyramid's `request_param="canvas_file"`

Having it named probably has some documentation benefit.

Reply:
Not yet, as it's used in `is_configured` below...

Reply:
If you look at the way this has turned out, we have sort of gone this way, but with our own. I'm sure there's an extra step here to get the rest of the way, but I think it might be to do away with the predicates for the most part.